Add note about bubbling (#68185)
vercel/next.js: docs/02-app/01-building-your-application/01-routing/05-error-handling.mdx

}
```

> You can bubble up errors to the next parent error boundary by throwing while rendering the `Error` component.

Reviewer: ```suggestion
If you want errors to bubble up to the parent error boundary, you can `throw` when rendering the `error` component.
```
